### PR TITLE
Added Speed Variable

### PR DESCRIPTION
**Pull request by noxwyll**
Maybe be done a bit crudely.

### CODE REVIEW

**Comment by avail** [comment]
indentation! tabs vs spaces!

File: Assets/Cubed's Unity Shaders/Shaders/Flat Lit Toon Rainbow.shader [hunk 1]
```diff
@@ -88,7 +91,7 @@ Shader "CubedParadox/Flat Lit Toon Rainbow" {
 ////// Lighting:
                 float attenuation = LIGHT_ATTENUATION(i);
                 float4 _MainTex_var = tex2D(_MainTex,TRANSFORM_TEX(i.uv0, _MainTex));
-                float4 node_1939 = _Time + _TimeEditor;
+                float4 node_1939 = _Time*_Speed + _TimeEditor;
```
Comment (avail) [inline Assets/Cubed's Unity Shaders/Shaders/Flat Lit Toon Rainbow.shader]: also put a space here (`(_Time * _Speed) + _TimeEditor`)

**Comment by noxwyll** [comment]
updated as requested.

**Comment by cubedparadox** [comment]
IMPORTANT: This shader is deprecated, I'm accepting this pull request for now, but will be replacing the rainbow shader with an updated variant in the future, built ontop of the latest toon shader and with a few new rainbow settings.

**Pull request merged**